src/routes/spellen/[game].svelte

```diff
@@ -84,7 +84,7 @@
 	function deletedHandler(event) {
 		const { success } = event.detail;
 		if (success) {
-			message = 'Spel opgeslagen in Mijn Gymles';
+			message = 'Spel verwijdert uit Mijn Gymles';
```

Comment:
```suggestion
			message = 'Spel verwijderdtdtdtdtdt uit Mijn Gymles';
```

Nee grapje, is verleden tijd, dus is `verwijderd`